### PR TITLE
Apply naming recommendations to all snippets and samples

### PR DESCRIPTION
**Pull request by donat**

### CODE REVIEW

**Comment by donat** [comment]
@bot-gradle test this

**Comment by donat** [comment]
@bot-gradle test this

**Comment by donat** [comment]
@bot-gradle test this

File: subprojects/docs/src/docsTest/java/org/gradle/docs/samples/ExemplarExternalSamplesFunctionalTest.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 package org.gradle.docs.samples;
 
+import org.gradle.integtests.fixtures.DonatLocalInitScriptModifier;
```
Comment (jjohannes) [inline subprojects/docs/src/docsTest/java/org/gradle/docs/samples/ExemplarExternalSamplesFunctionalTest.java:19]: I guess this sneaked in :)

**Review by jjohannes — CHANGES_REQUESTED** [review]
👍

**Review by jjohannes — APPROVED** [review]

**Pull request merged**